api_tests/meetings/views/test_meetings_detail.py

```diff
@@ -29,15 +29,15 @@ def user(self):
     @pytest.fixture()
     def meeting_submission_one(self, meeting, user):
         submission = ProjectFactory(title='Submission One', is_public=True)
-        submission.add_tag(meeting.endpoint, Auth(user))
+        meeting.submissions.add(submission)
```

Comment:
I could add a method to the Conference model that relates the node to the meeting (as is happening here) and then adds a log on the Node (currently, the only indicator is a log that a tag is added).  

I hesitated because I was unsure of desired behavior as this field is probably temporary (and Conference model likely to change drastically).